src/main.py

```diff
@@ -14,7 +14,8 @@
 # The user can enter a list of nameservers with the --ns param
 parser.add_argument('--ns', type=str, nargs='+', help="A list of NS servers to check.")
 parser.add_argument('--json', help="Output results from checks as JSON", action='count', default=0)
-
+# add argument to enable ipv6
+parser.add_argument('--ipv6', help ="Enables ipv6")
```

Comment:
Make it work the same way as `--json` for consistency. That entering only `--ipv6` enables it. Without `--ipv6` it defaults to ipv4. (no true or false value required for it)